__tests__/js-joda.test.ts

```diff
@@ -0,0 +1,16 @@
+import { LocalDate } from '@js-joda/core';
+import JsJodaUtils from "../packages/js-joda/src";
+
+describe("js-joda", () => {
+  const jsJodaUtils = new JsJodaUtils();
+
+  it("Should get the time difference between two LocalDates", () => {
```

Comment:
Can you add the same test for all the adapters?

Reply:
Done. Please let me know if that's what you were wanting.